### PR TITLE
V&V case composition-dependent model (Sandia Jet)

### PR DESCRIPTION
**Pull request by sjhbtue**
This pull request contains the files of the V&V case for the composition-dependent model, regarding configuration, grid convergence study and postprocessing. This work has been part of my Bachelor Final Project at Eindhoven University of Technology, where @bigfooted  and @Cristopher-Morales  were my supervisors.

### CODE REVIEW

**Comment by pcarruscag** [comment]
Don't forget to add the regression test to the vandv.py script.

**Comment by pcarruscag** [comment]
When are you planning to add the regression test?

**Comment by sjhbtue** [comment]
> When are you planning to add the regression test?

@pcarruscag I've created a PR for the TestCases repository. But I'm struggling with the PR for the SU2 repository. I've added the required piece of code in the vandv.py script (incorrect values for the residuals), created a folder for the species_transport and added the cfg file. I was trying to do the same as @Cristopher-Morales did before with getting the residuals by temporarily changing the branch in the regression.yml file in line 131, but this gets rejected. As a result, I'm struggling to get the values for the residuals for the vandv.py . Do you perhabs have an idea how I can deal with this issue?

**Comment by sjhbtue** [comment]
> That looks like an issue with authentication, I can't help with that

I'm looking into it with Cristopher

File: rans/SANDIA_jet/VVSandiaJet.cfg [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+MARKER_EULER= ( top_wall )
+MARKER_INLET_TURBULENT= ( inlet_gas, 0.04, 5, inlet_air, 0.004, 1 )
+%
+INC_INLET_TYPE=  VELOCITY_INLET VELOCITY_INLET
+SPECIFIED_INLET_PROFILE= NO
+%INLET_FILENAME= inlet_jet.dat
+%INLET_MATCHING_TOLERANCE= 1.2E-06
+%
+MARKER_INLET= ( inlet_gas, 294, 53.0, 1.0, 0.0 0.0, inlet_air, 294, 9.2, 1.0, 0.0, 0.0 )
+SPECIES_USE_STRONG_BC= NO
```
Comment (Cristopher-Morales) [inline rans/SANDIA_jet/VVSandiaJet.cfg:69]: I think you should remove this line

**Pull request merged**